practice_app/pages/api/getProductCatalog.test.js

```diff
@@ -0,0 +1,341 @@
+const getProductSet = async (req) => {
```

Comment:
This function is too long.

Also is this the actual implementation? It doesn't belong to a test file.